docs/build/apps/wallet/sep24.mdx

```diff
@@ -542,6 +701,12 @@ final paymentResult = await sdk.submitTransaction(transaction);
 
 Where `keypair` is the SEP-10 authenticated account. If you want to transfer funds from a different address, refer to [Changing Stellar Transfer Account](#changing-stellar-transfer-account) section.
 
+Code for submitting transactions to Stellar should be developed thoughtfully. The SDF has a documentation page dedicated to [submitting transactions and handling errors gracefully]. Here are a few things you need to keep in mind:
```

Comment:
missing a link?

Reply:
@piyalbasu the link is here at the bottom: https://github.com/stellar/stellar-docs/pull/930/files#diff-39e9cfa3d8b8714200fde6146f37f172556d4f1b750a333621353ac9dbd122feR788